test/integ/routers/alpha-router/alpha-router.integration.test.ts

```diff
@@ -1140,87 +1098,28 @@ describe('alpha router integration', () => {
                 recipient: alice._address,
                 slippageTolerance: SLIPPAGE,
                 deadline: parseDeadline(360),
-                simulate: { fromAddress: WHALES(tokenIn) },
+                simulate: { fromAddress: '0xeaf1c41339f7D33A2c47f82F7b9309B5cBC83B5F' },
```

Comment:
this change looks rather arbitrary

Reply:
this test is checking to make sure simulation is not called if the user does not have sufficient token balance. The test will fail without this change because the old address had sufficient balance